pkg/vrraw/vrraw.go

```diff
@@ -0,0 +1,49 @@
+// Package vrraw offers aliases to all VR abbreviations as defined here:
+//
+// http://dicom.nema.org/medical/dicom/current/output/html/part05.html#table_6.2-1
+//
+// Deprecated VRs from older editions are also included
+package vrraw
+
+const (
+	ApplicationEntity   = "AE"
```

Comment:
A thought occurred to me: these are all exported constants, therefore should have a comment describing the symbol.

As discussed in [Effective Go](https://golang.org/doc/effective_go.html#commentary):
 > Every exported (capitalized) name in a program should have a doc comment.

It might seem a bit much to do this, but I am actually thinking it could be nice documentation to have here and may save some folks a google search or trip to the [standard](http://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_6.2) (though they'll probably be doing that anyway haha).

If you have the time, maybe we can include a short one liner description for these (based on the standard description) and include a link to the relevant portion of the standard: http://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_6.2

I can also follow up and do this in another PR (I won't block the PR on this--just thought I'd mention).

Reply:
I don't think it seems a bit much at all, and is an excellent idea. Will add and push.